deb-get

```diff
@@ -1260,6 +1261,10 @@ function deb_deb-get() {
     SUMMARY="'apt-get' functionality for .debs published in 3rd party repositories or via direct download package."
 }
 
+#### MAIN ####
+
+export ${WGET_VERBOSITY:="-q"}
```

Comment:
```
# cannot do that as part of export - sorry my bad I think!
export CACHE_DIR="/var/cache/deb-get" WGET_VERBOSITY WGET_TIMEOUT
# and then get default values with something like
KLUDGE=${WGET_VERBOSITY:="-q"}
KLUDGE=${WGET_TIMEOUT:="--timeout 5"}
```

Reply:
correct. I should have double checked myself too. fixed now.